src/EventListener/LocaleListener.php

```diff
@@ -69,11 +69,12 @@ class LocaleListener implements EventSubscriberInterface
     /**
      * Construct the guessermanager
      *
-     * @param string               $defaultLocale  Framework default locale
-     * @param LocaleGuesserManager $guesserManager Locale Guesser Manager
-     * @param LoggerInterface      $logger         Logger
+     * @param string                 $defaultLocale     Framework default locale
+     * @param LocaleGuesserManager   $guesserManager    Locale Guesser Manager
+     * @param BestLocaleMatcher|null $bestLocaleMatcher Best Locale Matcher
+     * @param LoggerInterface|null   $logger            Logger
      */
-    public function __construct($defaultLocale = 'en', LocaleGuesserManager $guesserManager, BestLocaleMatcher $bestLocaleMatcher = null, LoggerInterface $logger = null)
+    public function __construct(string $defaultLocale, LocaleGuesserManager $guesserManager, BestLocaleMatcher $bestLocaleMatcher = null, LoggerInterface $logger = null)
```

Comment:
It should be a nullable string in the constructor, while on line #79 you should do $defaultLocale ?? 'en'